### PR TITLE
Update Korean Contributor

### PR DESCRIPTION
**Pull request by RieLCho**
I changed my GitHub ID 😃

### CODE REVIEW

**Comment by ZetaoYang** [comment]
@mmahmoudian Please review this and merge.

File: README.md [hunk 1]
```diff
@@ -40,4 +40,4 @@ If you need any help, feel free to [create an issue](https://github.com/flamesho
 
 ## Traditional Translation Way
 
-[Click here](https://github.com/flameshot-org/translation-instruction/blob/master/Traditional_Translation_Way.md).
\ No newline at end of file
+[Click here](https://github.com/flameshot-org/translation-instruction/blob/master/Traditional_Translation_Way.md).
```
Comment (mmahmoudian) [inline README.md]: What has changes here?!

Comment (mmahmoudian) [reply README.md]: can is be a bug in Github? I don't see any difference between those two lines.

Comment (RieLCho) [reply README.md]: ?! I didn't touch that line..... dang🤔

Comment (mmahmoudian) [reply README.md]: Let's merge it then as nothing has changed as far as I can see.

Comment (RieLCho) [reply README.md]: I guess... nothin' has changed 👀
Thank you.

**Pull request merged**